src/main/resources/view/MainWindow.fxml

```diff
@@ -3,54 +3,66 @@
 <?import javafx.scene.Scene?>
-<?import javafx.scene.control.Menu?>
-<?import javafx.scene.control.MenuBar?>
-<?import javafx.scene.control.MenuItem?>
-<?import javafx.scene.control.SplitPane?>
+<?import javafx.scene.control.Button?>
 <?import javafx.scene.image.Image?>
+<?import javafx.scene.image.ImageView?>
+<?import javafx.scene.layout.BorderPane?>
+<?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
+<?import javafx.stage.Stage?>
 
-<fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+<fx:root minHeight="600" minWidth="450" onCloseRequest="#handleExit" title="Rapportbook" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1">
   <icons>
-    <Image url="@/images/address_book_32.png" />
+    <Image url="@/images/rapportbook_32.png" />
   </icons>
   <scene>
     <Scene>
       <stylesheets>
-        <URL value="@DarkTheme.css" />
+        <URL value="@LightTheme.css" />
         <URL value="@Extensions.css" />
       </stylesheets>
 
       <VBox>
-        <MenuBar fx:id="menuBar" VBox.vgrow="NEVER">
-          <Menu mnemonicParsing="false" text="File">
-            <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit" />
-          </Menu>
-          <Menu mnemonicParsing="false" text="Help">
-            <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
-          </Menu>
-        </MenuBar>
+            <BorderPane prefHeight="51.0" prefWidth="340.0" styleClass="menu-bar">
+               <left>
+                  <Button fx:id="brandButton" mnemonicParsing="false" prefHeight="58.0" prefWidth="54.0" styleClass="logo-button" BorderPane.alignment="CENTER">
+                     <graphic>
+                        <ImageView fx:id="brandButtonImage" fitHeight="64.0" fitWidth="64.0" pickOnBounds="true" preserveRatio="true" styleClass="logo-button">
+                           <image>
+                              <Image url="@../images/brand_circle_icon.png" />
+                           </image>
+                        </ImageView>
+                     </graphic>
+                  </Button>
+               </left>
+               <right>
+                  <HBox alignment="TOP_RIGHT" prefHeight="100.0" prefWidth="200.0" BorderPane.alignment="CENTER">
+                     <children>
+                        <Button maxHeight="1.7976931348623157E308" mnemonicParsing="false" styleClass="menu-bar-nav-button" onAction="#handleHelp" text="Help" />
+                        <Button maxHeight="1.7976931348623157E308" mnemonicParsing="false" styleClass="menu-bar-nav-button" onAction="#handleExit" text="Exit" />
+                     </children>
+                  </HBox>
+               </right>
+            </BorderPane>
```

Comment:
Correct me if I'm wrong, but I think this chunk is indented more than it should be. I think it should be aligned with the deleted <MenuBar> section.

Reply:
Alright, I'll take a look and get back to you later.